testapi.pm

```diff
@@ -971,11 +973,12 @@ sub assert_script_run {
             # not change default timeout.
             timeout      => 90,
             fail_message => '',
-            quiet        => testapi::get_var('_QUIET_SCRIPT_CALLS')
+            quiet        => testapi::get_var('_QUIET_SCRIPT_CALLS'),
+            max_interval => undef
```

Comment:
I think this is the line that mangles the input (https://travis-ci.org/github/os-autoinst/os-autoinst/builds/691702877#L852)